src/main/resources/fingerprint-rules.yml

```diff
@@ -30,6 +28,9 @@ rules:
         label: true
       - predicate: SUBJECT
         label: true
+      - predicate: LANGUAGE
+        edgeProperties:
+          CODE: http://bibfra.me/vocab/marc/code
```

Comment:
Let us use the URI "http://bibfra.me/vocab/lite/language" for language code in fingerprint. "http://bibfra.me/vocab/marc/code" will not indicate that the code is for language.

`CODE: http://bibfra.me/vocab/lite/language`

Reply:
Done